### ISSUE TITLE
Can I annotate the EBUI module while I do inference?

### ISSUE THREAD

**Issue by JiuqingDong** [JosephKJ/OWOD#117]
I trained the OWOD and got the checkpoint.

Can I just annotate this line then compare the final result?
["classes = self.update_label_based_on_energy(logits, classes)"](https://github.com/JosephKJ/OWOD/blob/23890f188cd1a6801c6ac0e3dacd78b8572b8c29/detectron2/evaluation/pascal_voc_evaluation.py#L122)

I mean, when you compair these result, you train the model again, or just annotate the code of EBUI ? In my view, the EBUI is designd for post-processing.
<img width="662" alt="image" src="https://user-images.githubusercontent.com/111500217/233387557-2a6eff9f-fbd9-48b8-bbb0-d9d17680a64f.png">
